frontend/src/app/core/current-user/current-user.service.ts

```diff
@@ -75,7 +75,7 @@ export class CurrentUserService {
   /**
    * Returns the set of capabilities for the given context and/or actions
    */
-  public capabilities$(actions:string[] = [], projectContext:string|null = null):Observable<ICapability[]> {
+  public capabilities$(actions:string[] = [], projectContext:string|null):Observable<ICapability[]> {
```

Comment:
In this case we might as well work with `undefined` instead of `null`, I just figured it'd be better to make the "unset" option more explicit.

Reply:
yes indeed both would work, but I kind of liked an explicit "null" more. WDYT?

Reply:
What I mean is that if you remove the defaulting to null, then we are automatically using `undefined`, so if we want to accept `null` as a value here, then I'd argue it should default to null

Reply:
Ah you're absolutely right. I didn't see the first argument having a default